package.json

```diff
@@ -11,6 +11,7 @@
     "lint": "gulp lint",
     "test": "gulp test --browsers PhantomJS",
     "updated": "lerna updated --only-explicit-updates",
+    "prettier": "node prettier.js",
```

Comment:
I'd rather call this `format` to be more consistent with our `package.json` commands else where but as long as it exists I'm good.